<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>1280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="frame_error">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>1280</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame_error{background-color: rgba(0, 0, 0, 0.5);}</string>
   </property>
   <widget class="QWidget" name="we" native="true">
    <property name="geometry">
     <rect>
      <x>124</x>
      <y>468</y>
      <width>552</width>
      <height>345</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#we{image: url(:/1登录界面/Images/1登录界面/弹窗BG.png);}
</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>130</y>
       <width>552</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>用户名或密码错误，请重新输入！</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_yes">
     <property name="geometry">
      <rect>
       <x>206</x>
       <y>230</y>
       <width>140</width>
       <height>40</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{border-image: url(:/2系统首页/Images/2系统首页/弹窗/确--认-拷贝.png);}
QPushButton:pressed{border-image: url(:/2系统首页/Images/2系统首页/弹窗/确--认-拷贝-2.png);}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>139</width>
       <height>39</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_exit">
     <property name="geometry">
      <rect>
       <x>485</x>
       <y>40</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{border-image: url(:/2系统首页/Images/2系统首页/弹窗/关--闭-拷贝.png);}
QPushButton:pressed{border-image: url(:/2系统首页/Images/2系统首页/弹窗/关--闭-拷贝-按下.png);}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>29</width>
       <height>29</height>
      </size>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_input">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>1280</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame_input{background-image: url(Images/登录界面/BG.png);}
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>186</x>
      <y>843</y>
      <width>428</width>
      <height>72</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#widget_2{image: url(:/1登录界面/Images/1登录界面/圆角矩形-1.png);}</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>80</width>
       <height>72</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image:url(:/1登录界面/Images/1登录界面/mima-(2).png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_password">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>0</y>
       <width>350</width>
       <height>72</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>30</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>IBeamCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{color: rgb(255, 255, 255);
background:transparent;
border-style:none;}

</string>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhPreferLowercase|Qt::ImhSensitiveData</set>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>请输入您的密码</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_1" native="true">
    <property name="geometry">
     <rect>
      <x>186</x>
      <y>690</y>
      <width>428</width>
      <height>72</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">#widget_1{image: url(:/1登录界面/Images/1登录界面/圆角矩形-1.png);}</string>
    </property>
    <widget class="QLabel" name="label_1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>80</width>
       <height>72</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image:url(:/1登录界面/Images/1登录界面/用户名.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_userName">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>0</y>
       <width>350</width>
       <height>72</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>30</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>IBeamCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{color: rgb(255, 255, 255);
background:transparent;
border-style:none;}







</string>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhPreferLowercase</set>
     </property>
     <property name="text">
      <string>admin</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>请输入您的用户名</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_login">
    <property name="geometry">
     <rect>
      <x>289</x>
      <y>983</y>
      <width>222</width>
      <height>72</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>40</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::TabFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border:0;
background-color: rgb(13, 225, 255);
border-radius:10px;</string>
    </property>
    <property name="text">
     <string>登录</string>
    </property>
   </widget>
  </widget>
  <zorder>frame_input</zorder>
  <zorder>frame_error</zorder>
 </widget>
 <tabstops>
  <tabstop>lineEdit_userName</tabstop>
  <tabstop>lineEdit_password</tabstop>
  <tabstop>pushButton_login</tabstop>
  <tabstop>pushButton_yes</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
